src/aiperf/records/records_manager.py

```
@@ -1726,6 +1397,17 @@ async def _process_results(
             except Exception as e:
                 self.exception(f"Failed to publish server metrics results: {e!r}")
 
+        accuracy_enabled = (
+            self.run.cfg.accuracy is not None and self.run.cfg.accuracy.enabled
+        )
+        if accuracy_enabled and phase == CreditPhase.PROFILING:
+            try:
+                await self._publish_accuracy_results(phase)
+            except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)